brave/brave-core
# Move data from ledger_state into preferences

Resolves https://github.com/brave/brave-browser/issues/9869
Resolves https://github.com/brave/brave-browser/issues/9750
Resolves https://github.com/brave/brave-browser/issues/7165

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
plan 1:
- start browser from mater
- enable rewards
- set values in AC settings
- close the browser
- switch to this PR
- start the browser
- go to AC settings and make sure that all are the same
- go rewards internals and make sure that wallet info matches as well

plan 2:
- enable rewards
- claim grant
- make sure that you see total balance with rates correctly

plan 3:
- enable rewards
- go to unverified publisher and open tip and monthly banner. Make sure that you see default values (currently 1, 10, 100) in there

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
